src/features/commands/premium.js

```diff
@@ -64,11 +64,11 @@ module.exports = {
                     })
             }
         } catch (e) {
+            console.log(e);
             return interaction.reply({
                 content: `Unfortunately, an error occured: \"${e.message}\"`,
```

Comment:
Don't put unsanitized errors in messages

Reply:
What should we say if people want to refer to error though.

Reply:
Look at the suggested change

Reply:
LOL why didn't it show up.

Reply:
github website on mobile is buggy, so submitting the review then editing worked